src/Knp/Component/Pager/Exception/PagerBadArgumentException.php

```diff
@@ -0,0 +1,22 @@
+<?php
```

Comment:
I don't think we need a custom exception

Reply:
Yes it's too much for that small case